src/screens/Home/Home.tsx

```diff
@@ -38,6 +39,50 @@ interface HomeProps {
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
 
+  const calculateCustomerData = () => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let toCharge = 0;
+    let outstandingPayments = 0;
+    let customers = currentSite.customers;
+    for (let i = 0; i < customers.length; i++) {
+      const currCustomer = customers[i]
+      //depends if the meter readings list is sorted with earliest => latest
+      let latestMeterReading = currCustomer.meterReadings[currCustomer.meterReadings.length - 1];
+      if (latestMeterReading != null) {
+        let lessMonth = lessThanCurrentMonth(latestMeterReading.date)
+        if (lessMonth) {
+          toCharge += 1;
+        }
+        if (checkPayment(currCustomer)) {
```

Comment:
This 1-liner I think makes the code harder to read. I don't know what "checking" a payment means from this context (and I have to look into the function to figure it out). We should either improve the function name so that it's clear what this `if` statement actually checks, or, since the function is a 1-liner, is simple, and only used here, we could remove the function and do the logic inside of the if statement (separating it into 2 lines for clarity would help).